### PR TITLE
chore: move quickstart pages

### PR DESCRIPTION
**Pull request by octonato**
References #724

### CODE REVIEW

**Review by raboof — APPROVED** [review]

**Review by johanandren — APPROVED** [review]

**Comment by octonato** [comment]
As mentioned in https://github.com/lightbend/akkaserverless-docs/pull/1124
The pages have now fully moved to the SDK and the few references to documents in the main docs site are now using absolute links.

File: docs/src/modules/java/pages/quickstart/cr-value-entity-scala.adoc [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+* The package name, `customer.api`.
+* Import `google/protobuf/empty.proto`.
++
+[source,proto,indent=0]
+.src/main/proto/customer/api/customer_api.proto
+----
+include::example$scala-customer-registry-quickstart/src/main/proto/customer/api/customer_api.proto[tag=declarations]
+----
+
+. Add the service endpoint. The service endpoint is annotated with `akkaserverless.codegen` indicating we want to generate a Value Entity for this service.
```
Comment (johanandren) [inline docs/src/modules/java/pages/quickstart/cr-value-entity-scala.adoc:104]: Old annotation name, or should that stay?

File: docs/src/modules/java/pages/quickstart/cr-value-entity-views-java.adoc [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
++
+[source,proto,indent=0]
+.src/main/proto/customer/view/customer_view.proto
+----
+include::example$java-customer-registry-views-quickstart/src/main/proto/customer/view/customer_view.proto[tag=CustomerByEmail]
+----
+<1> The Protobuf `service` for the View.
+<2> The option that the Maven plugin will use to generate the `CustomerByEmail` View.
+<3> The `UpdateCustomer` method defines how Akka Serverless will update the view.
+<4> The source of the View is the `"customers"` Value Entity. This identifier is defined in the `entity_type: "customers"` property of the `(akkaserverless.file).value_entity` option in the `customer_domain.proto` file.
```
Comment (johanandren) [inline docs/src/modules/java/pages/quickstart/cr-value-entity-views-java.adoc:89]: No longer correct?

**Review by johanandren — COMMENTED** [review]
Looks like the guide text contents needs some updates to the new annotations as well?

File: docs/src/modules/java/pages/quickstart/cr-value-entity-scala.adoc (hunk 1 above)
Comment (octonato) [reply docs/src/modules/java/pages/quickstart/cr-value-entity-scala.adoc:104]: What do you mean? This is about the new annotation. Code snippet is also showing the new.

Comment (octonato) [reply docs/src/modules/java/pages/quickstart/cr-value-entity-scala.adoc:104]: hmm, I just spotted a left over but in another file. Maybe you commented on the wrong line?

**Pull request merged**